Apps.DeepL/Actions/GetUsageAction.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Apps.DeepL.Requests;
+using Apps.DeepL.Responses;
+using Blackbird.Applications.Sdk.Common;
+using Blackbird.Applications.Sdk.Common.Actions;
+using Blackbird.Applications.Sdk.Common.Invocation;
+using Blackbird.Applications.SDK.Extensions.FileManagement.Interfaces;
+using DeepL;
+
+namespace Apps.DeepL.Actions;
+
+[ActionList]
+public class GetUsageActionActions : DeepLInvocable
+{
+
+    public GetUsageActionActions(InvocationContext invocationContext) : base( invocationContext )
+    {
+
+    }
+
+    [Action("GetUsage", Description = "Get usage information")]
```

Comment:
Action name should follow the same naming convention as a description. So it should be "Get usage"